contrib/pylightning/lightning/plugin.py

```diff
@@ -99,7 +99,7 @@ class Plugin(object):
 
     """
 
-    def __init__(self, stdout=None, stdin=None, autopatch=True):
+    def __init__(self, stdout=None, stdin=None, autopatch=True, dynamic=True):
```

Comment:
dynamic needs to default to False, I think.  Though perhaps there are only a few plugins which can simply be fixed.  @cdecker ?

Reply:
All plugins from the repo seems ok (even with `dynamic = True` they are not restarted.

Reply:
Should be ok for now, I'll fixup the various plugins in `lightningd/plugins` once this gets merged.